cypress/e2e/spec.cy.js

```diff
@@ -1,5 +1,18 @@
-describe('template spec', () => {
-  it('passes', () => {
-    cy.visit('https://example.cypress.io')
+//Test number 1
+describe('Gallery app', () => {
+  it('User can make chart as expected', () => {
+    cy.visit('http://localhost:8080/')
```

Comment:
You should be able to just use "/" here, because the baseUrl should have been set in cypress.config.js. Not critical but just a note for future ease of use.